pkgs/modules/python/default.nix

```diff
@@ -25,16 +25,23 @@ let
     inherit pypkgs;
   };
 
+  #  poetry-config = pkgs.writeTextFile {
+  #    name = "poetry-config";
+  #    text = ''
+  #      [[tool.poetry.source]]
+  #      name = "replit"
+  #      url = "https://package-proxy.replit.com/pypi/simple/"
+  #      default = true
+  #
+  #    '';
+  #    destination = "/conf.toml";
```

Comment:
I don't think we should leave this here commented out, because it might trick people into thinking conf.toml does something.

Reply:
👍

The contents are effectively the same as what's in our pip wrapper, so if needed they can be pulled out of git history or that file.